@types/index.d.ts

```diff
@@ -3,10 +3,16 @@
 /* eslint-disable no-var, import/no-mutable-exports */
 
 import {Agent} from 'http';
-import {AbortSignal} from 'abort-controller';
-import Blob from 'fetch-blob';
+import * as Blob from 'fetch-blob';
```

Comment:
Maybe the best thing to do is to import with `require`, as `fetch-blob` is a CommonJS library? 🤔 That should work without any user having to have synthetic imports or es module interop enabled...

```suggestion
import Blob = require('fetch-blob');
```

Reply:
Both methods should work with no `esModuleInterop` nor synthetic imports.
```typescript
import * as Blob from 'fetch-blob';
import Blob = require('fetch-blob');
```

I prefer the `*` import personnally but I can change it if you prefer.

Reply:
In the last commit, I forced `esModuleInterop` and `allowSyntheticDefaultImports` to `false` in tsd config to ensure that it works correctly and it seems to.